assignments/this.js

```diff
@@ -1,10 +1,18 @@
-/* The for principles of "this";
+/* The four principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. Whenever a function is contained in the global scope, the
+value of ${this} inside that function will be the window object.
```

Comment:
To be a little more general, in the global scope, the value of `this` would be the _global object_. What that object specifically is depends on the environment. In browsers, it is indeed the `window` object, but in Node.js, a JavaScript runtime used to run servers and other applications with JavaScript, the global object would be `console`.